x-pack/legacy/plugins/siem/public/components/open_timeline/open_timeline.tsx

```diff
@@ -4,15 +4,32 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { EuiPanel } from '@elastic/eui';
-import React from 'react';
-
+import { EuiPanel, EuiContextMenuPanel, EuiContextMenuItem, EuiBasicTable } from '@elastic/eui';
+import React, { useMemo, useCallback, useRef } from 'react';
+import { FormattedMessage } from '@kbn/i18n/react';
 import { OPEN_TIMELINE_CLASS_NAME } from './helpers';
 import { OpenTimelineProps } from './types';
 import { SearchRow } from './search_row';
 import { TimelinesTable } from './timelines_table';
 import { TitleRow } from './title_row';
 
+import * as i18n from './translations';
+import { useStateToaster } from '../toasters';
+import { TimelineDownloader } from './export_timeline/export_timeline';
+import { DeleteTimelineModalButton } from './delete_timeline_modal';
+import {
+  UtilityBarGroup,
+  UtilityBarText,
+  UtilityBar,
+  UtilityBarSection,
+  UtilityBarAction,
+} from '../utility_bar';
+export interface ExportTimelineIds {
+  timelineId: string | null | undefined;
```

Comment:
can we have a `timelineId` null or undefined when we tried to export timelines? I think the two others attributes can default to an empty array. That might simplify some code.

Reply:
Yes, I'm going to remove `ExportTimelineIds` as we just need timelineId as `string` when exporting timeline